lib/galaxy/config/sample/job_conf.xml.sample_advanced

```diff
@@ -457,7 +457,7 @@
         <!-- Destinations define details about remote resources and how jobs
              should be executed on those remote resources.
          -->
-        <destination id="local" runner="local"/>
+          <destination id="local" runner="local" />
```

Comment:
Some of the indentation changes seem wrong to me, maybe use `xmllint` to reformat it?

Reply:
Not sure what vscode did there, plain old sed did the trick now.